src/lib/POWERMGNT/POWERMGNT.cpp

```diff
@@ -193,27 +193,28 @@ PowerLevels_e POWERMGNT::setPower(PowerLevels_e Power)
     switch (Power)
     {
     case PWR_10mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 600);  
+        analogWrite(GPIO_PIN_RFamp_APC2, 600);
         break;
     case PWR_25mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 770);  
+        analogWrite(GPIO_PIN_RFamp_APC2, 770);
         break;
-    case PWR_50mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 950);  
+    case PWR_100mW:
+        analogWrite(GPIO_PIN_RFamp_APC2, 1150);
+        CurrentPower = PWR_100mW;
         break;
     case PWR_250mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 1480); 
+        analogWrite(GPIO_PIN_RFamp_APC2, 1480);
         break;
     case PWR_500mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 2000); 
+        analogWrite(GPIO_PIN_RFamp_APC2, 2000);
         break;
     case PWR_1000mW:
-        analogWrite(GPIO_PIN_RFamp_APC2, 3500); 
+        analogWrite(GPIO_PIN_RFamp_APC2, 3500);
         break;
-    case PWR_100mW:
+    case PWR_50mW:
     default:
-        analogWrite(GPIO_PIN_RFamp_APC2, 1150);  
-        CurrentPower = PWR_100mW;
+        analogWrite(GPIO_PIN_RFamp_APC2, 950);
+        CurrentPower = PWR_50mW;
```

Comment:
`Power = PWR_50mW` because `CurrentPower = Power;` later :)

Reply:
Sorry, i missed that earlier :/

Reply:
Good catch. Also removed an unneeded set that was a copy/paste error on 100mw